docs/getting-started/windows-manual.md

```diff
@@ -72,3 +84,22 @@ Start-Service splunk-otel-collector
 ```
 
 The collector logs and errors can be viewed in the Windows Event Viewer.
+
+### Docker
+
+Create build of otel collector on local system.
+```bash
+$ git clone https://github.com/signalfx/splunk-otel-collector.git
+$ cd splunk-otel-collector
+$ docker build -t otelcol --build-arg OTEL_MSI_VERSION=0.31.0 -f .\cmd\otelcol\Dockerfile.windows .
```

Comment:
@jcheng-splunk would a `quay.io/signalfx/splunk-otel-collector-windows` be warranted here instead of requiring users to build their own?

Reply:
> @jcheng-splunk would a `quay.io/signalfx/splunk-otel-collector-windows` be warranted here instead of requiring users to build their own?

Yes, we'll need to build/tag/push the image from our CI/CD pipeline.